vms/platformvm/validators/fee/fee_test.go

```diff
@@ -222,9 +221,9 @@ var (
 				MinPrice:                 minPrice,
 				ExcessConversionConstant: excessConversionConstant,
 			},
-			expectedSeconds: 11,
-			expectedCost:    math.MaxUint64, // Should not overflow
-			expectedExcess:  math.MaxUint32 * 11,
+			expectedSeconds: 10,
```

Comment:
Why is `expectedSeconds` reducing? I expected the changes in test values to be non-negative because this new logic finds the _last_ time point at which they can pay the fee.

Reply:
Previously we returned the first time that our balance went 0 or negative. Now we return the last time that we have a non-negative balance. So the tests should either be unchanged (if the value were hitting 0 exactly) or be reduced by one (which is the case here).

Reply:
Ahh gotcha; I forgot what the previous behaviour was.